exercises/change/change_test.py

```diff
@@ -19,8 +19,8 @@ def test_change_with_Lower_Elbonia_Coins(self):
                          [21, 21, 21])
 
     def test_large_target_values(self):
-        self.assertEqual(find_minimum_coins(21, [2, 5, 10, 20, 50]),
-                         [2, 2, 2, 5, 10])
+        self.assertEqual(find_minimum_coins(219, [1, 5, 10, 20, 50]),
```

Comment:
Will it be possible to update values to match `canonical-data.json` for the exercise? - https://github.com/exercism/problem-specifications/blob/master/exercises/change/canonical-data.json

```
    {
      "description": "large target values",
      "property": "findFewestCoins",
      "coins": [1, 2, 5, 10, 20, 50, 100],
      "target": 999,
      "expected": [   2,   2,   5,  20,  20
                  ,  50, 100, 100, 100, 100
                  , 100, 100, 100, 100, 100 ]
    },
```